frontend/src/utilities/useWatchNotebookEvents.tsx

```diff
@@ -35,7 +35,7 @@ export const useWatchNotebookEvents = (
               notification.error('Error fetching notebook events', e.response.data.message);
               clear();
             });
-          watchHandle = setTimeout(watchNotebookEvents, FAST_POLL_INTERVAL);
+          watchHandle = setTimeout(watchNotebookEvents, POLL_INTERVAL_NOTEBOOK_EVENTS);
```

Comment:
I wonder if it is not better to just setTimeout when we finish -- instead of increasing the timeout. If we go with server side pagination, we'll end up likely chewing up all this time paginating on the server and the client won't send another request (at least for Chrome) until the first request finishes... which may end up in a series of `getEvent` calls waiting the larger the number of events are.

Reply:
Yup, you're right, i think it's a great change, ill get on that tomorrow morning

Reply:
Do we really care about events older than say... 200?  Is it worth paginating for these?  I'm not sure we would really want to retrieve the last 100000 events, anyway, would we?  We certainly don't want to be stuck paging through them.  I've run into this in some weird cases and I can see there being a lot of events for a pod that comes back to life everytime there's a notebook

Reply:
This is one of those places where using the watcher and websocket communication would bring the best of react. It would add too much complexity for a quick fix, but maybe we can check this kind of use case for the pass through API, it would be viable?

Reply:
Taling about that, I don't see the perks of using pagination in this case. We always want the latest events, adding pagination will only mean that we will have multiple requests if there are too many events, it would make sense if we could sort the request by a timestamp, so we can request more data if the latest chunk is newer than the timestamp of the creation of the notebook. But right now I was trying some configs and the only thing I achieved was multiple calls to fetch the same data.

Reply:
I've tested the folowing case:
Add pagination in the server, only store the events that start after the notebook creation and return the call to the frontend once we reach the limit of the pagination. With this method we have *N* calls in the backend and only one in the frontend, I think it can release pressure from the list, but it's still a heavy call from the frontend, cause it will mean that it might be waiting several kube calls to display the values. Time wise, with this little data I'm not seeing a significant difference between no limiting pagination and limiting pagination and storing all the values till returning something, not sure if releasing that change to the PR.